### DISCUSSION TITLE
Вопрос по процессу design-review

### DISCUSSION THREAD

**Discussion by Lootjs** [chocofamilyme/choco-ui#34] [Q&A] [answered]
Пока не очень понимаю как наладить процесс дизайн-ревью входящих PR. 
Есть сторибук, но он синхронизирован только с веткой main. 
Вряд ли дизайнер сможет стягивать ветки , запускать сторибук локально.
Идеи?

**Comment by abdraymd** [comment]
Деплоить отдельно ветки как в РБ? После ревью очищать

**Reply by Lootjs** [reply]
нет инфры, куда можно было бы заливать ветки. у нас я деплою в нашу виртуалку.

**Comment by Lootjs** [comment, marked as answer]
Впрочем, думаю можно сделать интеграцию с netlify/heroku либо что то из похожего

**Reply by KirillSolovyev** [reply]
Можно создать ветку `develop` или `testing` и создавать PR в эту ветку. После того, как ветка прошла ревью мерджить. При мердже запускать хук, чтобы деплоить эту ветку куда-нибудь.

**Reply by Lootjs** [reply]
Ветка develop станет ботлнеком. Как минимум, из за медленного дизайн ревью.
К тому же, когда уже смерджено, разрабам менее резонно фиксить что компонент, мне кажется. 
Пока думаю деплоить каждую ветку в нетлифай

**Reply by KirillSolovyev** [reply]
Если deploy будет происходит автоматически, то решение отличное :+1:

**Reply by Lootjs** [reply]
#37 . ветки (и main) деплоятся в netlify, в комментах будет ссылка.

**Reply by Lootjs** [reply]
но теперь вопрос, кому из дизайнеров проводить дизайн ревью. Предлагаю, проектам обращаться к своим дизайнерам, в случае если их нет, то в чате просим глянуть главного дизайнера 🤔
